include/image.mk

```diff
@@ -554,7 +554,25 @@ define Device/Build/image
 
   $(BIN_DIR)/$(call IMAGE_NAME,$(1),$(2)): $(KDIR)/tmp/$(call IMAGE_NAME,$(1),$(2))
 	cp $$^ $$@
-
+	$(if $(CONFIG_CREATE_JSON), \
+		DEVICE_ID="$(DEVICE_NAME)" \
+		TOPDIR="$(TOPDIR)" \
+		BIN_DIR="$(BIN_DIR)" \
+		IMAGE_NAME="$(IMAGE_NAME)" \
+		IMAGE_TYPE=$(word 1,$(subst ., ,$(2))) \
+		IMAGE_SIZE="$(IMAGE_SIZE)" \
+		DEVICE_TITLE="$(DEVICE_TITLE)" \
+		DEVICE_VENDOR="$(DEVICE_VENDOR)" \
+		DEVICE_MODEL="$(DEVICE_MODEL)" \
+		DEVICE_VARIANT="$(DEVICE_VARIANT)" \
```

Comment:
I would suppress an empty VARIANT in json file instead of the `variant: ""` in the commit message.
So, one would need an IF either here or in the code processing it.

Reply:
Why suppress it? Reading it from a any other script would cause an empty string "" instead of a key error, what do you think?